beavy/common/morphing_schema.py

```diff
@@ -16,3 +16,16 @@ def _obj_to_name(self, obj):
     def _get_serializer(self, obj):
         name = self._obj_to_name(obj)
         return self.registry.get(name, self.FALLBACK)()
+
+
+class MorphingNested(fields.Nested, MorphingSchema):
+
+    def _get_serializer(self, obj):
+        name = self._obj_to_name(obj)
+        rv = self.nested.registry.get(name, self.nested.FALLBACK)()
+        return rv
+
+    def _serialize(self, nested_obj, attr, obj):
+        rv = [self._get_serializer(obj).dump(obj).data
+                    for obj in nested_obj]
```

Comment:
I'm not sure at all that this is sufficient for the wider range of cases that this function might have to handle. Thoughts?